iceoryx_posh/include/iceoryx_posh/internal/popo/base_client.hpp

```diff
@@ -129,16 +130,16 @@ class BaseClient
     ///
     /// @brief const accessor of the underlying port
     ///
-    const Port& port() const noexcept;
+    const PortT& port() const noexcept;
```

Comment:
Not a must but if you have `PortType` at your disposal why not use it - reads better.

Reply:
I think it is easier to use in the `inl` file since `PortT` is the template parameter which is available in the `inl` file but for `PortType` I would need to use `typename BaseClient<PortT, TriggerHandleT>::PortType`